### DISCUSSION TITLE
Welcome to neonize Discussions!

### DISCUSSION THREAD

**Discussion by krypton-byte** [krypton-byte/neonize#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kashishjain02** [comment]
where can i find the documentation for neonize

**Comment by Hendroarifin** [comment]
which hosting supports neonize? I've tried pythonanywhere but it doesn't work, I also tried replit and it doesn't work. Even though I run it on a local computer it can run normally

**Comment by jindalshreyansh** [comment]
Suggest you should make a proper documentation. Atleast for basics like sending text to a contact, send attachment
